integration-tests/structured-logging/__tests__/to-do.js

```diff
@@ -76,7 +77,8 @@ const toMatchSchema = (received, schema) => {
     }
   } else {
     return {
-      message: () => validationResult.error,
+      message: () =>
+        `${validationResult.error}\n\n${JSON.stringify(received, null, 2)}`,
```

Comment:
This is just so output mention what the input to be validate was. Without it we were getting just: `ValidationError: "action" does not match any of the allowed types` without knowing what `action` actually looks like